src/CppUTest/TestRegistry.cpp

```diff
@@ -50,6 +50,7 @@ void TestRegistry::runAllTests(TestResult& result)
     result.testsStarted();
     for (UtestShell *test = tests_; test != NULL; test = test->getNext()) {
```

Comment:
yeah, I also plan to do that. and what's your thinking of interface segregate for UtestShell? maybe We need to do it in case UtestShell interface become fatter.